HiddenThread.user.js

```diff
@@ -22,272 +22,36 @@ const SIGNED_POST_TYPE = 1;
-                y = (this.mt[kk] & UPPER_MASK) | (this.mt[kk + 1] & LOWER_MASK);
-                this.mt[kk] = this.mt[kk + (M - N)] ^ (y >>> 1) ^ mag01[y & 1];
-            }
-
-            y = (this.mt[N - 1] & UPPER_MASK) | (this.mt[0] & LOWER_MASK);
-            this.mt[N - 1] = this.mt[M - 1] ^ (y >>> 1) ^ mag01[y & 1];
-            this.mti = 0;
-        }
-
-        y = this.mt[this.mti++];
-
-        y ^= (y >>> 11);
-        y ^= (y << 7) & 0x9d2c5680;
-        y ^= (y << 15) & 0xefc60000;
-        y ^= (y >>> 18);
-
-        return y >>> 0;
-    };
-
-    /**
-     * Generates a random unsigned 31-bit integer.
-     *
-     * @since 0.1.0
-     * @returns {number}
-     */
-    MersenneTwister.prototype.int31 = function () {
-        return this.int() >>> 1;
-    };
-
-    /**
-     * Generates a random real in the interval [0;1] with 32-bit resolution.
-     *
-     * @since 0.1.0
-     * @returns {number}
-     */
-    MersenneTwister.prototype.real = function () {
-        return this.int() * (1.0 / (MAX_INT - 1));
-    };
-
-    /**
-     * Generates a random real in the interval ]0;1[ with 32-bit resolution.
-     *
-     * @since 0.1.0
-     * @returns {number}
-     */
-    MersenneTwister.prototype.realx = function () {
-        return (this.int() + 0.5) * (1.0 / MAX_INT);
-    };
-
-    /**
-     * Generates a random real in the interval [0;1[ with 32-bit resolution.
-     *
-     * @since 0.1.0
-     * @returns {number}
-     */
-    MersenneTwister.prototype.rnd = function () {
-        return this.int() * (1.0 / MAX_INT);
-    };
-
-    /**
-     * Generates a random real in the interval [0;1[ with 32-bit resolution.
-     *
-     * Same as .rnd() method - for consistency with Math.random() interface.
-     *
-     * @since 0.2.0
-     * @returns {number}
-     */
-    MersenneTwister.prototype.random = MersenneTwister.prototype.rnd;
-
-    /**
-     * Generates a random real in the interval [0;1[ with 53-bit resolution.
-     *
-     * @since 0.1.0
-     * @returns {number}
-     */
-    MersenneTwister.prototype.rndHiRes = function () {
-        var a = this.int() >>> 5,
-            b = this.int() >>> 6;
-
-        return (a * 67108864.0 + b) * (1.0 / 9007199254740992.0);
-    };
-
-    var instance = new MersenneTwister();
-
-    /**
-     * A static version of [rnd]{@link module:MersenneTwister#rnd} on a randomly seeded instance.
-     *
-     * @static
-     * @function random
-     * @memberof module:MersenneTwister
-     * @returns {number}
-     */
-    MersenneTwister.random = function () {
-        return instance.rnd();
-    };
+const injectLib = (url) => {
+    let lib = document.createElement("script")
+    lib.type = "text/javascript"
+    lib.src = url
+    document.head.appendChild(lib)
+}
 
-    return MersenneTwister;
-}));
+injectLib("https://cdnjs.cloudflare.com/ajax/libs/jszip/3.6.0/jszip.min.js")
+injectLib("https://cdn.jsdelivr.net/npm/mersennetwister@0.2.3/src/MersenneTwister.min.js")
 

```

Comment:
Можно elliptic.js добавить с гитхаба:
```suggestion
injectLib("https://raw.githubusercontent.com/indutny/elliptic/43ac7f230069bd1575e1e4a58394a512303ba803/dist/elliptic.min.js")
```